docs/en/06_customising_add_search_service.md

```diff
@@ -47,7 +47,8 @@ in all these implementations that the array of documents is appropriately sized.
 
 All methods that rely on API calls should throw `IndexingServiceException` on error.
 
-### addDocument(DocumentInterface $document): self
+
+### addDocument(string $indexSuffix, DocumentInterface $document): ?string
 
 This method is responsible for adding a single document to the indexes. Keep in mind, the
```

Comment:
Does this line still hold true? Mentions adding document to the indexes, but a single index is now being passed in.